Healthy/Classes/Utilities/Logger/NewRelicLogger.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import NewRelic
```

Comment:
You need to add it to the loggers in `Classes/Utilities/Logger/Logger.swift`

Reply:
Please let me know if it's hard for you to apply the integration.

Reply:
![Screenshot 2023-06-22 at 1 47 55 PM](https://github.com/motoon-eg/healthy/assets/87352168/7241c86a-1e6b-4c10-8b55-afa9a0ac679b)
![Screenshot 2023-06-22 at 1 48 04 PM](https://github.com/motoon-eg/healthy/assets/87352168/c30eb3eb-7a2c-4a9b-99fb-2b0570296c55)

#### is that the integration you look for ?

Reply:
and there is another thing , we could use `NRLogger` and as you told me:  "NewRelic recodes events by default" , so we didn't have to create the `NewRelicLogger` class 
we can just `import NewRelic` in our `Logger.swift` and use it like  : 

![Screenshot 2023-06-22 at 2 21 00 PM](https://github.com/motoon-eg/healthy/assets/87352168/299c6bc3-ad2b-4e7a-b894-f7644245dfee)

Reply:
instead of using logger, you should use loggsers: [Logger] and use the new relic wrapper as a logger...

Reply:
Then you pass the events to the all loggers instead...